Chalenges/fizzBuzz/fizz-buzz.js

```diff
@@ -0,0 +1,54 @@
+class Node {
+  constructor(value) {
+    this.value = value;
+    this.left = null;
+    this.right = null;
+  }
+}
+
+class FizzBuzzTree {
+  constructor() {
+    this.root = null;
+  }
+
+  add(thing) {
+    let newNode = new Node(thing, null, null); 
+  
+    if(this.root === null) {
+      this.root = newNode;
+    } else {
+      addMotif(this.root, newNode);
+    }
+  }
+  fizzBuzz() {
+    const preArr = [];
+    const start = this.root;
+    fizzBuzzMotif(start, preArr);
+    return;
+  }
+
+}
+//I couldn't figure out recursion on my own so I referenced Dave Trost's code for this challenge
```

Comment:
again, thanks so much for noting this. good job on taking a reference and adapting it to your own code!